README.md

```diff
@@ -1377,7 +1377,9 @@ ANSWERS.addComponent('Map', {
   // Optional, configuration for the map's behavior when a query returns no results
   noResults: {
     // Optional, whether to display all results in the vertical when no results are found. Defaults to false, in which case only the no results card will be shown.
```

Comment:
Is this documentation correct? It references a no results card. That won't be shown by the Map component.

Reply:
Do we also need to update the "this._noResults" block in the constructor?

Reply:
updated the readme and the this._noResults block in mapcomponent.js's constructor. I wasn't sure if it made sense to be explicit about 'visible' in that file, since it only gets used in mapprovider.js not mapcomponent.js, but on second thought I don't know where else it would go.